editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1035,6 +1035,29 @@ void CanvasItemEditor::_viewport_input_event(const InputEvent &p_event) {
 
 		const InputEventMouseButton &b = p_event.mouse_button;
 
+		if (b.button_index == GESTURE_MAGNIFY) {
+
+			if (zoom < MIN_ZOOM) {
```

Comment:
The code form here on is really similar than:
```c++
b.button_index == BUTTON_WHEEL_DOWN
b.button_index == BUTTON_WHEEL_UP
```
would it be worth it to make a zoom inline function?
we could remove redundant code. and it would be easier to understand.